pkg/bgpv1/agent/controller.go

```
@@ -95,17 +91,21 @@ type ControlPlaneState struct {
 	IPv6 net.IP
 }
 
+type policyLister interface {
+	List() []*v2alpha1api.CiliumBGPPeeringPolicy
```

Why avoid the possibility to return an error? Multiple implementations could exist, would it be possible that one might want to return an error at some point?